openapi.yaml

```diff
@@ -85,6 +85,8 @@ paths:
     $ref: paths/workflow_runs.yaml#/tasks
   /workflow_runs/{workflow_run_id}/tasks/{task_id}:
     $ref: paths/workflow_runs.yaml#/task
+  /workflow_runs/{workflow_run_id}/tasks/{task_id}/complete:
+    $ref: paths/workflow_runs.yaml#/task  
```

Comment:
`/complete` is missing at the end, right?

Reply:
The `/complete` only applies to the endpoint URL. Line 89 is the path reference to where the endpoint is described in the workflow_runs.yaml file